### ISSUE TITLE
[Question] Could we register new polygons or new geometric form 

### ISSUE THREAD

**Issue by emmachuche** [scaleflex/filerobot-image-editor#255]
I want to know it's possible to register a new geometric form inside the draw section

HERE:
![image](https://user-images.githubusercontent.com/19750665/186295924-a9e8af10-d35f-4c47-be1d-100a43962c27.png)

**Comment by SaraZoarob** [comment]
Hi @emmachuche, can you clarify more which shape you want to add and we will check the possibility to add it

**Comment by emmachuche** [comment]
> Hi @emmachuche, can you clarify more which shape you want to add and we will check the possibility to add it

Yes! i want to add a new polygon for example a "check icon" and remove for example the pen option

**Comment by rainabozhi** [comment]
plus one on that question- but it would also be helpful if we could draw polygons with different lengths for the sides. 
so now if I stretch a polygon or a rectangle it stretches on both sides- is there a way to stretch a single side?
